src/test/java/touro/chess/chessNotationTest.java

```diff
@@ -0,0 +1,18 @@
+package touro.chess;
+
+import org.junit.After;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class chessNotationTest {
```

Comment:
This needs to match the name of the class. It should be called `LocationTest`.

Also remember, class names ALWAYS begin with an upper case letter.